roles/openshift_metrics/tasks/create_temp_dirs.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create temp directory for doing work in on target
+  command: mktemp -td openshift-metrics-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+
+- name: Create temp directory for all our templates
+  file: path={{mktemp.stdout}}/templates state=directory mode=0755
+  changed_when: False
+  when: metrics_create_template_dir | bool
+
+- name: Create temp directory local on control node
+  local_action: command mktemp -d
+  register: local_tmp
+  changed_when: False
+  vars:
+    ansible_become: false
```

Comment:
Not sure if this is necessary, but its fine to keep it

Reply:
not sure either, but I want to preserve the metrics config.yaml playbook as much as possible.